python/fledge/plugins/south/person_detection/videostream.py

```diff
@@ -55,6 +55,12 @@ def __init__(self, resolution=(640, 480), framerate=30, source=0, enable_thread=
             _ = self.stream.set(4, resolution[1])
         elif detectCoralDevBoard():
             self.stream = cv2.VideoCapture(source)
+            # cannot change camera resolution on coral device through opencv API.
+        else:
+            # If the device is not supported self.stream would be None.
+            # There could be another format such as YUYV which may support changing camera resolution
+            # through Open CV API. Omitting these calls for now.
+            self.stream = cv2.VideoCapture(source)
```

Comment:
should be = None ? (per comments)

Reply:
A better way to handle the error is as follows 

```
cap = VideoCapture(0)
while True:
    ret, frame = cap.read()
    if ret:
        cv2.imshow('window', frame)
```

If nothing is there `ret` will be False.